src/main/java/net/ossindex/version/impl/BoundedVersionRange.java

```diff
@@ -148,6 +148,20 @@ public String toString()
     throw new UnsupportedOperationException("Cannot get string for range");
   }
 
+  @Override
+  public String toMavenString()
+  {
+    if (range != null) {
+      return "[" + minimum + "," + maximum + "]";
+    }
+    if (minimum != null) {
```

Comment:
Why not `if (minimum != null && maximum == null)` ?

Reply:
@buddysaul Above range != null condition will handle (minimum != null && maximum == null) condition

Reply:
I think this would be clearer like this:
`if (maximum != null || minimum == null) {
  throw new UnsupportedOperationException();
}
return "[" + minimum.toString() + "]";`

Reply:
@buddysaul  Updated Condition. Please review